smk-edit/static/components/edit-item-details-layer-esri-dynamic.js

```diff
@@ -4,6 +4,49 @@ export default importComponents( [
 ] ).then( function () {
     return vueComponent( import.meta.url, {
         props: [ 'itemId' ],
+        computed: {
+            hasLabels: function() {
+                const layer = this.$store.getters.configLayer( this.itemId );
```

Comment:
You don't have to fix, but consider choosing a 'style' and being consistent with respect to whitespace. I'm not a fan of all the whitespace on the smk/smk-cli code base, but not a deal breaker. For example, either always have space before a '(' or never have a space (or follow JSHint defaults).
Same for content within a set of parentheses, either always have whitespace or never have whitespace. I find consistency makes for more readable code. 

There are no hard and fast rules of course, just something to think about.

Reply:
I agree with consistency. Maybe we can set a preference for JSHint.